app/browser/tabs.js

```diff
@@ -1001,6 +1013,16 @@ const api = {
         if (isSessionPartition(createProperties.partition)) {
           createProperties.parent_partition = ''
         }
+        if (createProperties.isTor) {
```

Comment:
we only need to keep `createProperties.parent_partition = ''` 
other attributes are redundant because `GuestViewManager::CreateGuest` doesn't read them and no need to

Thanks @riastradh-brave for asking question